src/wovn_interceptor.php

```diff
@@ -68,6 +68,10 @@
         }
         $headers->responseOut();
 
+        if (!$headers->canProcessResponse()) {
+            return $buffer;
+        }
```

Comment:
This stops WOVN.php from adding hreflangs and modifying the response. When you are using a default lang alias with path pattern, `/page.php` should not be handled at all.